src/batched/KokkosBatched_Util.hpp

```diff
@@ -43,6 +43,12 @@ namespace KokkosBatched {
 #pragma message "KOKKOS_BATCHED_DISABLE_AVX512"
 #endif
 
+#if defined(__FMA__)
+#pragma message "KOKKOS_BATCHED_ENABLE_FMA"
```

Comment:
@ambrad @kyungjoo-kim @srajama1 - guys, you know that even with all this front end handling, the compiler may choose to fuse intrinsics/instructions during the final code generation/optimisation passes? The only to really stop FMAs being generated is to use `-fno-fma` I think which tells the code generation it is not permitted to create any instructions of this type. Just want you to know what you might end up running evening if it isn't explicitly encoded.